non_iid_detection/requirements.txt

```
@@ -0,0 +1,7 @@
+# This notebook currently requires the latest developer version of cleanlab: pip install git+https://github.com/cleanlab/cleanlab.git 
+numpy==1.22.0
+pandas==1.4.1
+scipy==1.7.3
+scikit_learn==1.0.2
```

timm is missing. Pick a version. I think 0.6.13 works

```suggestion
scikit_learn==1.0.2
timm==X.Y.Z 
```

---

I've added timm as a dependency.